packages/@n8n/config/src/configs/logging.config.ts

```diff
@@ -44,4 +55,19 @@ export class LoggingConfig {
 
 	@Nested
 	file: FileLoggingConfig;
+
+	/**
+	 * Scopes to filter logs by. Disabled by default.
```

Comment:
Maybe to be more explicit?
```suggestion
	 * Scopes to filter logs by. Nothing is filtered by default.
```